validation/citrus-validation-hamcrest/src/main/java/org/citrusframework/validation/matcher/hamcrest/HamcrestValidationMatcher.java

```diff
@@ -125,7 +124,8 @@ public void validate(String fieldName, String value, List<String> controlParamet
      * @return
      */
     private Matcher<?> getMatcher(String matcherName, String[] matcherParameter, TestContext context) {
-        try {
+
+       try {
```

Comment:
that is something minor:

```suggestion
        try {
```